sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitStmt.cs

```diff
@@ -1201,12 +1201,12 @@ private void VisitFloatingLiteral(FloatingLiteral floatingLiteral)
         var outputBuilder = StartCSharpCode();
         if (floatingLiteral.ValueString.EndsWith(".f", StringComparison.Ordinal))
         {
-            outputBuilder.Write(floatingLiteral.ValueString.AsSpan()[..^1]);
```

Comment:
The `.AsSpan()` here is important, it avoids unnecessarily allocating for the common cases, which helps keep GC pressure down and performance up